core/src/rpc.rs

```diff
@@ -145,29 +159,32 @@ impl JsonRpcRequestProcessor {
     }
 
     pub fn get_inflation(&self, commitment: Option<CommitmentConfig>) -> Result<Inflation> {
-        Ok(self.bank(commitment).inflation())
+        Ok(self.bank(commitment)?.inflation())
     }
 
     pub fn get_epoch_schedule(&self) -> Result<EpochSchedule> {
         // Since epoch schedule data comes from the genesis config, any commitment level should be
         // fine
-        Ok(*self.bank(None).epoch_schedule())
+        Ok(*self
+            .bank(None)
+            .expect("working_bank must exist")
```

Comment:
`.bank(None)` doesn't return the working_bank AFAICT, so this `.expect("working_bank must exist")` seems off?   But in general how about just `?` instead of the `.expect()`.  It's a situation we don't expect to ever happen but if it does I'd rather the RPC call fail instead of exiting the entire process